h2o-r/h2o-package/R/automl.R

```diff
@@ -189,6 +190,9 @@ h2o.automl <- function(x, y, training_frame,
 
   # Update build_control list with top level build control args
   build_control <- list(stopping_criteria = list(max_runtime_secs = max_runtime_secs))
+  if (!is.null(max_runtime_secs_per_model)) {
+      build_control$stopping_criteria$max_runtime_secs_per_model <- max_runtime_secs_per_model
```

Comment:
@seb-h2o so we can use two notations interchangeably `self.build_control["stopping_criteria"]["max_runtime_secs_per_model"]` and  `build_control$stopping_criteria$max_runtime_secs_per_model` ?

Reply:
yep